nip57.ts

```diff
@@ -13,7 +13,7 @@ export function useFetchImplementation(fetchImplementation: any) {
   _fetch = fetchImplementation
 }
 
-export async function getZapEndpoint(metadata: Event): Promise<null | string> {
+export async function getZapEndpoint(metadata: Event<0>): Promise<null | string> {
```

Comment:
This is the only potentially breaking change. TypeScript users will have to update their code to: `getZapEndpoint(event as Event<0>)`.

Once we update relay.ts to infer types from filters (in a later MR) this shouldn't be necessary.